discovery/eureka/client.go

```diff
@@ -97,6 +97,7 @@ func fetchApps(ctx context.Context, server string, client *http.Client) (*Applic
 		resp.Body.Close()
 	}()
 
+	//nolint:usestdlibvars
```

Comment:
I think we can open an issue on usestdlibvars for this.
I'd be a good improvement.

Reply:
Thanks, I agree it would be best if they fixed this case. I'll open an issue with them.

Reply:
@machine424 I can reproduce the problem with `usestdlibvars` v1.26.0, but it's apparently fixed in v1.27.0 (the latest).